zqd/ingest/pcap.go

```diff
@@ -109,6 +109,7 @@ func (p *Process) run(ctx context.Context) error {
 		os.Remove(p.space.DataPath(PcapIndexFile))
 		os.Remove(p.space.DataPath(space.AllBzngFile))
 		p.space.SetPacketPath("")
+		p.space.UnsetTimes()
```

Comment:
Because this pr added some logic missing in the existing `space.Info`, this had to be added in order to fix unit tests.